bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html

```diff
@@ -21,18 +21,15 @@
   <h2 class="moss-section-heading">{{ _('About') }}</h2>
 
   <p class="moss-copy">
-  {% trans url1=url('mozorg.about.manifesto') %}
-    The Global Mission Partners: India track is open to any open source or
-    free software project that is undertaking an activity which significantly
-    furthers Mozilla’s <a href="{{ url1 }}">mission</a>, and for which the
-    applicant(s) are based in India.
+  {% trans url1=url('mission-partners.html') %}
```

Comment:
This is causing the page to throw an error:

```
Reverse for 'mission-partners.html' not found. 'mission-partners.html' is not a valid view function or pattern name.
```

```suggestion
  {% trans url1=url('mozorg.moss.mission-partners') %}
```